components/social_links.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { PropTypes } from 'react';
+import _ from 'lodash';
+
+import twitterIcon from '../images/icons/twitter.svg';
+import githubIcon from '../images/icons/github.svg';
+import dribbbleIcon from '../images/icons/dribbble.svg';
+
+const Icons = {
+  twitter: twitterIcon,
+  github: githubIcon,
+  dribbble: dribbbleIcon,
+};
+
+export default class SocialLinks extends React.Component {
```

Comment:
True, but in my opinion it would be better to have consistency across projects, the usual pattern is the one that @fribmendes mentioned.